src/simplay-web/src/SetSpriteFrameEvent.ts

```diff
@@ -0,0 +1,14 @@
+import { SetSpriteFrameEventArgs } from './SetSpriteFrameEventArgs';
+import { Event } from './Event';
+import { EventAction } from './EventAction';
+
+export class SetSpriteFrameEvent extends Event {
+  readonly args: SetSpriteFrameEventArgs;
+  constructor(forId: string, timestamp: number, args: SetSpriteFrameEventArgs) {
+    super(forId, timestamp, EventAction.SET_SPRITE_FRAME, args);
+    this.args = args;
```

Comment:
All of these Event classes call `this.args = ...` is there a way we could move that to the parent class? 
I don't think so,but maybe you have an idea?

Reply:
I don't think so unfortunately, since we want to have args be initialized within the constructor...